public/i18n/en/translation.json

```diff
@@ -575,7 +575,15 @@
         "success_run_backup": "Backup performed successfully",
         "success_download": "Backup downloaded successfully",
         "date": "Date",
-        "backups": "Backups"
+        "backups": "Backups",
+        "octet-stream": "Encrypted",
+        "gzip": "Not encrypted",
+        "pgp-encrypted": "Encrypted",
+        "size": "Size",
+        "mimetype": "Type",
+        "delete": "Delete",
+        "modal_delete_title": "Delete remote backup",
```

Comment:
```suggestion
        "modal_delete_title": "Delete backup",
```

Reply:
Why removing the "delete" world?

Reply:
This is the only place where we use the term "remote"; if we want to focus on this concept I think we should have written "remote" in other places too :man_shrugging: We can leave it as it is anyway, it's not so important

Reply:
I agree, we should add the "remote" word in other labels